cmd/cert-checker/main.go

```diff
@@ -70,8 +70,9 @@ type certChecker struct {
 	pa           core.PolicyAuthority
 	dbMap        *gorp.DbMap
 	certs        chan core.Certificate
-	issuedReport report
 	clock        clock.Clock
+	rMu          *sync.Mutex
```

Comment:
Nit: It would be better for this to have a more descriptive name, e.g., `reportMutex`